boards/arm/nrf7001dk_nrf5340/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+# Copyright (c) 2023 Nordic Semiconductor ASA.
```

Comment:
Can we please modify the commit to log to add that this is based on nrf7002 copy.

Reply:
Done